src/pages/createAppointment/selectPost.tsx

```diff
@@ -0,0 +1,75 @@
+import { useContext, useEffect, useState } from "react";
+import { NextRouter, useRouter } from "next/router";
+import { userContext } from "supabase/user_context";
+import { useSupabaseClient } from "@supabase/auth-helpers-react";
+import { Database } from "supabase/db_types";
+import { Box, Typography } from "@mui/material";
+import Loading from "@/components/public/Loading";
+import Navbar from "@/components/public/Navbar";
+import { Post } from "@/types/Post";
+import { PagePaths } from "enum/pages";
+import { GetCurrentUserPosts } from "@/services/Posts";
+import { Grid } from "@mui/material";
+import EachPostCard from "@/components/createAppointment/EachPostCard";
+
+const MainLayout = {
+  margin: "1vh 0 0 0",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  width: "100%",
+  flexDirection: "row",
+  padding: "30px",
+};
+
+export default function ShowAppointment() {
+  const router: NextRouter = useRouter();
+  const userStatus = useContext(userContext);
+  const supabaseClient = useSupabaseClient<Database>();
+  const [posts, setPosts] = useState<Post[] | null>(null);
+
+  function handleClickCard(item: Post) {
+    router.push(PagePaths.createAppointment + "/" + item?.postId);
```

Comment:
`?` means it nullable while `item` obviously not null, so this isn't neccessary

Reply:
Fixed it.